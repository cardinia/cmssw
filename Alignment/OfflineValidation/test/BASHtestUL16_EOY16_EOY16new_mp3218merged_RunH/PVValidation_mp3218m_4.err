26-Sep-2019 13:15:54 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/DCFEFA24-1212-E911-9091-0CC47A4D76B6.root
26-Sep-2019 13:16:15 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/DCFEFA24-1212-E911-9091-0CC47A4D76B6.root
Begin processing the 1st record. Run 281641, Event 271892853, LumiSection 162 on stream 0 at 26-Sep-2019 13:19:23.779 CEST
Begin processing the 1001st record. Run 281641, Event 277255093, LumiSection 165 on stream 0 at 26-Sep-2019 13:44:26.744 CEST
Begin processing the 2001st record. Run 281641, Event 285110096, LumiSection 169 on stream 0 at 26-Sep-2019 13:49:29.613 CEST
Begin processing the 3001st record. Run 281641, Event 294076902, LumiSection 174 on stream 0 at 26-Sep-2019 13:54:14.139 CEST
Begin processing the 4001st record. Run 281641, Event 301189210, LumiSection 179 on stream 0 at 26-Sep-2019 13:58:54.413 CEST
Begin processing the 5001st record. Run 281641, Event 308201894, LumiSection 183 on stream 0 at 26-Sep-2019 14:03:16.527 CEST
Begin processing the 6001st record. Run 281641, Event 316178910, LumiSection 187 on stream 0 at 26-Sep-2019 14:07:58.767 CEST
Begin processing the 7001st record. Run 281641, Event 325325325, LumiSection 192 on stream 0 at 26-Sep-2019 14:12:47.944 CEST
Begin processing the 8001st record. Run 281641, Event 332032268, LumiSection 196 on stream 0 at 26-Sep-2019 14:17:20.251 CEST
Begin processing the 9001st record. Run 281641, Event 340428805, LumiSection 200 on stream 0 at 26-Sep-2019 14:22:09.610 CEST
Begin processing the 10001st record. Run 281641, Event 346690389, LumiSection 204 on stream 0 at 26-Sep-2019 14:26:26.384 CEST
Begin processing the 11001st record. Run 281641, Event 356780072, LumiSection 209 on stream 0 at 26-Sep-2019 14:30:50.565 CEST
Begin processing the 12001st record. Run 281641, Event 363408727, LumiSection 213 on stream 0 at 26-Sep-2019 14:35:14.831 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 14:37:32 CEST  Run: 281641 Event: 367155507
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 1292
%MSG
Begin processing the 13001st record. Run 281641, Event 371461343, LumiSection 217 on stream 0 at 26-Sep-2019 14:39:43.636 CEST
Begin processing the 14001st record. Run 281641, Event 379379124, LumiSection 222 on stream 0 at 26-Sep-2019 14:44:07.729 CEST
Begin processing the 15001st record. Run 281641, Event 386888283, LumiSection 226 on stream 0 at 26-Sep-2019 14:48:36.797 CEST
Begin processing the 16001st record. Run 281641, Event 393998589, LumiSection 231 on stream 0 at 26-Sep-2019 14:52:48.656 CEST
Begin processing the 17001st record. Run 281641, Event 402606737, LumiSection 236 on stream 0 at 26-Sep-2019 14:56:34.168 CEST
Begin processing the 18001st record. Run 281641, Event 409493622, LumiSection 240 on stream 0 at 26-Sep-2019 15:00:23.365 CEST
Begin processing the 19001st record. Run 281641, Event 418218238, LumiSection 245 on stream 0 at 26-Sep-2019 15:04:17.720 CEST
Begin processing the 20001st record. Run 281641, Event 425789003, LumiSection 250 on stream 0 at 26-Sep-2019 15:08:15.192 CEST
Begin processing the 21001st record. Run 281641, Event 434049284, LumiSection 255 on stream 0 at 26-Sep-2019 15:12:25.239 CEST
Begin processing the 22001st record. Run 281641, Event 441324505, LumiSection 259 on stream 0 at 26-Sep-2019 15:16:28.326 CEST
Begin processing the 23001st record. Run 281641, Event 449249342, LumiSection 264 on stream 0 at 26-Sep-2019 15:20:31.428 CEST
Begin processing the 24001st record. Run 281641, Event 456354356, LumiSection 268 on stream 0 at 26-Sep-2019 15:24:39.953 CEST
Begin processing the 25001st record. Run 281641, Event 464043526, LumiSection 273 on stream 0 at 26-Sep-2019 15:28:47.554 CEST
Begin processing the 26001st record. Run 281641, Event 471705903, LumiSection 278 on stream 0 at 26-Sep-2019 15:33:02.065 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 15:37:07 CEST  Run: 281641 Event: 479480505
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 7111
%MSG
Begin processing the 27001st record. Run 281641, Event 479350500, LumiSection 282 on stream 0 at 26-Sep-2019 15:37:23.893 CEST
Begin processing the 28001st record. Run 281641, Event 487482500, LumiSection 293 on stream 0 at 26-Sep-2019 15:41:06.730 CEST
Begin processing the 29001st record. Run 281641, Event 496316832, LumiSection 300 on stream 0 at 26-Sep-2019 15:45:05.087 CEST
Begin processing the 30001st record. Run 281641, Event 503435736, LumiSection 305 on stream 0 at 26-Sep-2019 15:49:38.829 CEST
Begin processing the 31001st record. Run 281641, Event 511722688, LumiSection 310 on stream 0 at 26-Sep-2019 15:54:17.465 CEST
Begin processing the 32001st record. Run 281641, Event 520042326, LumiSection 315 on stream 0 at 26-Sep-2019 15:58:47.600 CEST
26-Sep-2019 16:02:13 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/DCFEFA24-1212-E911-9091-0CC47A4D76B6.root
26-Sep-2019 16:02:13 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/0A0B8917-1B12-E911-A7D6-0CC47A78A3D8.root
26-Sep-2019 16:02:16 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/0A0B8917-1B12-E911-A7D6-0CC47A78A3D8.root
Begin processing the 33001st record. Run 281641, Event 1956224, LumiSection 2 on stream 0 at 26-Sep-2019 16:03:37.836 CEST
Begin processing the 34001st record. Run 281641, Event 8937372, LumiSection 7 on stream 0 at 26-Sep-2019 16:08:04.770 CEST
Begin processing the 35001st record. Run 281641, Event 18270492, LumiSection 13 on stream 0 at 26-Sep-2019 16:12:20.126 CEST
Begin processing the 36001st record. Run 281641, Event 25325887, LumiSection 18 on stream 0 at 26-Sep-2019 16:16:57.763 CEST
Begin processing the 37001st record. Run 281641, Event 33729532, LumiSection 24 on stream 0 at 26-Sep-2019 16:21:41.008 CEST
Begin processing the 38001st record. Run 281641, Event 41265350, LumiSection 28 on stream 0 at 26-Sep-2019 16:26:12.113 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 16:27:04 CEST  Run: 281641 Event: 42933367
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 2298
%MSG
Begin processing the 39001st record. Run 281641, Event 49308009, LumiSection 32 on stream 0 at 26-Sep-2019 16:30:49.021 CEST
Begin processing the 40001st record. Run 281641, Event 55951012, LumiSection 36 on stream 0 at 26-Sep-2019 16:35:13.210 CEST
Begin processing the 41001st record. Run 281641, Event 65655857, LumiSection 41 on stream 0 at 26-Sep-2019 16:39:30.993 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 16:41:21 CEST  Run: 281641 Event: 67085980
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 339
%MSG
Begin processing the 42001st record. Run 281641, Event 72344758, LumiSection 45 on stream 0 at 26-Sep-2019 16:43:36.390 CEST
Begin processing the 43001st record. Run 281641, Event 80322678, LumiSection 49 on stream 0 at 26-Sep-2019 16:47:33.751 CEST
Begin processing the 44001st record. Run 281641, Event 88011041, LumiSection 54 on stream 0 at 26-Sep-2019 16:51:47.625 CEST
Begin processing the 45001st record. Run 281641, Event 95436762, LumiSection 58 on stream 0 at 26-Sep-2019 16:55:42.637 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 16:59:40 CEST  Run: 281641 Event: 103742215
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 0
%MSG
Begin processing the 46001st record. Run 281641, Event 104526715, LumiSection 63 on stream 0 at 26-Sep-2019 16:59:42.784 CEST
Begin processing the 47001st record. Run 281641, Event 110314938, LumiSection 67 on stream 0 at 26-Sep-2019 17:03:36.340 CEST
Begin processing the 48001st record. Run 281641, Event 120007885, LumiSection 72 on stream 0 at 26-Sep-2019 17:07:21.025 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 17:09:49 CEST  Run: 281641 Event: 124513976
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 247
%MSG
Begin processing the 49001st record. Run 281641, Event 126600279, LumiSection 76 on stream 0 at 26-Sep-2019 17:11:13.474 CEST
26-Sep-2019 17:15:09 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/0A0B8917-1B12-E911-A7D6-0CC47A78A3D8.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 OutOfBounds          -w TrackRefitter:Fi                       6        6
    2 fileAction           -s file_close                             2        2
    3 fileAction           -s file_open                              4        4

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 OutOfBounds          281641/367155507 281641/479480505 281641/124513976
    2 fileAction           PostProcessEvent PostGlobalEndRun 
    3 fileAction           pre-events       pre-events       PostProcessEvent

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
Warning                 6                   6
System                  6                   6

dropped waiting message count 0
[10.06MB/10.06MB][100%][==================================================][10.06MB/s]  [10.06MB/10.06MB][100%][==================================================][10.06MB/s]  
