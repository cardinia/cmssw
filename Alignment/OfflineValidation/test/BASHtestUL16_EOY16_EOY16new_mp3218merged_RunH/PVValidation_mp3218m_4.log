000 (3189599.004.000) 09/26 11:25:39 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3189599.004.000) 09/26 12:50:45 Started transferring input files
	Transferring to host: <188.185.210.225:9618?addrs=188.185.210.225-9618+[2001-1459-301-95--100-db]-9618&noUDP&sock=16410_1454_55524>
...
040 (3189599.004.000) 09/26 12:50:45 Finished transferring input files
...
001 (3189599.004.000) 09/26 12:50:45 Job executing on host: <188.185.210.225:9618?addrs=188.185.210.225-9618+[2001-1459-301-95--100-db]-9618&noUDP&sock=16277_3f22_3>
...
006 (3189599.004.000) 09/26 12:50:54 Image size of job updated: 880
	1  -  MemoryUsage of job (MB)
	880  -  ResidentSetSize of job (KB)
...
006 (3189599.004.000) 09/26 12:55:55 Image size of job updated: 14592
	11  -  MemoryUsage of job (MB)
	10768  -  ResidentSetSize of job (KB)
...
006 (3189599.004.000) 09/26 13:00:55 Image size of job updated: 37784
	32  -  MemoryUsage of job (MB)
	32532  -  ResidentSetSize of job (KB)
...
006 (3189599.004.000) 09/26 13:05:55 Image size of job updated: 60684
	50  -  MemoryUsage of job (MB)
	50476  -  ResidentSetSize of job (KB)
...
006 (3189599.004.000) 09/26 13:10:56 Image size of job updated: 103520
	86  -  MemoryUsage of job (MB)
	87880  -  ResidentSetSize of job (KB)
...
006 (3189599.004.000) 09/26 13:15:57 Image size of job updated: 162108
	123  -  MemoryUsage of job (MB)
	125032  -  ResidentSetSize of job (KB)
...
006 (3189599.004.000) 09/26 13:20:57 Image size of job updated: 445000
	399  -  MemoryUsage of job (MB)
	408436  -  ResidentSetSize of job (KB)
...
006 (3189599.004.000) 09/26 13:25:57 Image size of job updated: 457884
	412  -  MemoryUsage of job (MB)
	421232  -  ResidentSetSize of job (KB)
...
006 (3189599.004.000) 09/26 13:30:58 Image size of job updated: 464020
	418  -  MemoryUsage of job (MB)
	427356  -  ResidentSetSize of job (KB)
...
006 (3189599.004.000) 09/26 13:35:59 Image size of job updated: 469140
	423  -  MemoryUsage of job (MB)
	432400  -  ResidentSetSize of job (KB)
...
006 (3189599.004.000) 09/26 13:40:59 Image size of job updated: 593548
	543  -  MemoryUsage of job (MB)
	555892  -  ResidentSetSize of job (KB)
...
006 (3189599.004.000) 09/26 13:45:59 Image size of job updated: 599248
	550  -  MemoryUsage of job (MB)
	562408  -  ResidentSetSize of job (KB)
...
006 (3189599.004.000) 09/26 16:06:15 Image size of job updated: 599248
	566  -  MemoryUsage of job (MB)
	579252  -  ResidentSetSize of job (KB)
...
006 (3189599.004.000) 09/26 16:11:16 Image size of job updated: 601780
	570  -  MemoryUsage of job (MB)
	582812  -  ResidentSetSize of job (KB)
...
006 (3189599.004.000) 09/26 16:36:17 Image size of job updated: 603140
	570  -  MemoryUsage of job (MB)
	582812  -  ResidentSetSize of job (KB)
...
040 (3189599.004.000) 09/26 17:15:33 Started transferring output files
...
040 (3189599.004.000) 09/26 17:15:33 Finished transferring output files
...
005 (3189599.004.000) 09/26 17:15:33 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 03:22:20, Sys 0 00:00:21  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 03:22:20, Sys 0 00:00:21  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	11035  -  Run Bytes Sent By Job
	1015  -  Run Bytes Received By Job
	11035  -  Total Bytes Sent By Job
	1015  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :    10353        1   2080032 
	   Memory (MB)          :      570     2000      2000 
...
