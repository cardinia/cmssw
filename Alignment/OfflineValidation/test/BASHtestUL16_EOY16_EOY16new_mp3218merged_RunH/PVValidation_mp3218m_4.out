Job started at  Thu Sep 26 12:50:56 CEST 2019
batch dir: /pool/condor/dir_29644 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_281641_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 11M -rw-r--r--. 1 tkello zh 11M Sep 26 17:15 PVValidation_mp3218m_281641.root -rw-r--r--. 1 tkello zh 20K Sep 26 12:51 PVValidation_mp3218m_281641_cfg.py -rw-r--r--. 1 tkello zh 9.6K Sep 26 17:15 _condor_stderr -rw-r--r--. 1 tkello zh 522 Sep 26 12:59 _condor_stdout -rwxr-xr-x. 1 tkello zh 1015 Sep 26 12:50 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 26 13:15 tkello.cc drwx------. 3 tkello zh 4.0K Sep 26 12:50 tmp drwx------. 3 tkello zh 4.0K Sep 26 12:50 var
Job ended at  Thu Sep 26 17:15:33 CEST 2019
